config/crd/bases/configuration.konghq.com_kongplugins.yaml

```diff
@@ -38,13 +42,29 @@ spec:
       openAPIV3Schema:
         description: KongPlugin is the Schema for the kongplugins API
         properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
           config:
             description: Config contains the plugin configuration.
             type: object
             x-kubernetes-preserve-unknown-fields: true
           configFrom:
             description: ConfigFrom references a secret containing the plugin configuration.
             properties:
+              apiVersion:
+                description: 'APIVersion defines the versioned schema of this representation
+                  of an object. Servers should convert recognized schemas to the latest
+                  internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+                type: string
+              kind:
+                description: 'Kind is a string value representing the REST resource
+                  this object represents. Servers may infer this from the endpoint
+                  the client submits requests to. Cannot be updated. In CamelCase.
+                  More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+                type: string
```

Comment:
As discussed in the call: `apiVersion`, `kind` and `metadata` don't belong as fields of `configFrom`, and should not be present here.

Reply:
I missed one of the two in 007bf90913a39534c08add3bf24fad0e8f4f7b80. d1b554b8 fixes this for the type we use in KongPlugin.